src/parser/view_transform.c

```diff
@@ -1557,6 +1561,30 @@ mq_remove_select_list_for_inline_view (PARSER_CONTEXT * parser, PT_NODE * statem
       /* just add one of integer value */
       new_select_list = pt_make_integer_value (parser, 1);
     }
+  else if (PT_SPEC_IS_DERIVED (spec) && spec->info.spec.derived_table_type == PT_IS_SUBQUERY)
+    {
+      spec->info.spec.referenced_attrs = new_select_list;
+
+      for (col = new_select_list; col; col = col->next)
+	{
+	  attr_order = pt_find_node_order (parser, spec->info.spec.as_attr_list, col);
+	  if (attr_order != -1)
+	    {
+	      nth_node = pt_get_node_from_list (subquery->info.query.q.select.list, attr_order - 1);
+	      if (nth_node != NULL && PT_IS_ANALYTIC_NODE (nth_node))
```

Comment:
```suggestion
	      assert (nth_node != NULL);
	      if (PT_IS_ANALYTIC_NODE (nth_node))
```
PT_IS_ANALYTIC_NODE 내부에서 이미 NULL 체크가 이루어지고 있습니다.
이렇게 변경하는 것은 어떨까요?

Reply:
기존 코드에서 제안하신 방식을 자주 사용하는 것 같습니다.
제안하신대로 변경하겠습니다.